### PR TITLE
Update roadmap post-release

### PR DESCRIPTION
**Pull request by billygriffin**
This is not time sensitive. I did a few things here:

1. Added the things that were previously on the roadmap and shipped in 1.5 to a past category. I'm not wedded to it, but it might be a helpful way to have a place to come back to to revisit the features that shipped in the past few releases at any given time and evaluate success (especially if we have associated metrics).

2. Added upcoming areas of exploration to the roadmap to make it a bit more robust and communicative. Because a few of the things are earlier in the process there's not a ton of amplifying info (like Accessibility), but I think that may be ok. Open to other opinions.

3. Altered formatting somewhat to make it more readable to me (if others don't feel like it is, please say so).

### CODE REVIEW

File: docs/process/roadmap.md [hunk 1]
```diff
@@ -2,22 +2,43 @@
...
-
-### Support rebase workflows
+#### Support rebase workflows
 
 - Improve rebase workflows: [#5953](https://github.com/desktop/desktop/issues/5953)
 - Measuring success: TBD alongside the work + usability testing
+
+#### Accessibility improvements
+
+- To be further defined, but we want to prioritize scoping a chunk of work on improving accessibility for our users
```
Comment (say25) [inline docs/process/roadmap.md]: Are there something that are clearly in or out of scope? Internationalization, colors used or font used? 

I would imagine internationalization in particular might be highly requested but maybe too large to be part of this effort

Comment (shiftkey) [reply docs/process/roadmap.md]: > Are there something that are clearly in or out of scope? Internationalization, colors used or font used?

The primary focus here is that Desktop is well-supported with screenreaders like NVDA and VoiceOver - that users with accessibility needs are able to navigate and work with Desktop like everyone else.

Comment (say25) [reply docs/process/roadmap.md]: Sounds good. Just thought maybe we could be more specific to help those sorts of users vs color blind users

Comment (billygriffin) [reply docs/process/roadmap.md]: @say25 This one in particular will get more tightly scoped over the next 3 weeks.

**Pull request merged**